app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardGeneralStep/MappingWizardGeneralStep.js

```diff
@@ -11,14 +20,30 @@ const MappingWizardGeneralStep = props => (
       label={__('Name')}
       required
       component={FormField}
-      validate={[required({ msg: __('Required') })]}
       type="text"
+      help={__(nameHelp)}
+      validate={[
+        required({
+          msg: __(nameRequired)
```

Comment:
OH you translate here... maybe earlier comment is hogwash... but then again... is one location is better than the other 🤔

Reply:
ohh, `__()` is the `sprintf` you meant? I think I did it here only to prevent a line break in the constants file. I can move it back if that's more clear. I'll do that, actually.

Reply:
@AllenBW was this `__()` sufficient to address your hogwash? or should I still not be using string template literals to put these numbers in these strings?

Reply:
Thinking its better to do the translation in the constants file with sprintf yeah, its the same as `__()` but allows for addition of variables...

Reply:
Ok i'll look at sprintf. The commits I just pushed fix everything but that.